{
  "_annotation": {
    "cve_id": "CVE-2024-32648",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-32648.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:vyperlang:vyper:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "6DA7B40A-DC0C-5791-8559-A71211EE7648",
                "versionEndExcluding": "0.3.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}